tests/kafkatest/services/verifiable_producer.py

```diff
@@ -73,24 +73,27 @@ def __init__(self, context, num_nodes, kafka, topic, max_messages=-1, throughput
         self.acked_values = []
         self.not_acked_values = []
         self.produced_count = {}
-        self.prop_file = ""
+
+    def prop_file(self, idx):
+        prop_file = ""
+        self.security_config = self.kafka.security_config.client_config(self.prop_file)
```

Comment:
Yes I agree, it was already that way -- self.security_config was set in  _worker(). Maybe because we did not really support multiple producers until recently. I will move setting self.security_config to __init__()